clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -374,6 +374,8 @@ Changes in existing checks
 - Improved :doc:`readability-qualified-auto
   <clang-tidy/checks/readability/qualified-auto>` check by adding the option
   `AllowedTypes`, that excludes specified types from adding qualifiers.
+  Added the option `IgnoreAliasing`, that allows not looking at underlying
```

Comment:
Please rebase from `main` and reword entry - Release Notes were cleared after 21.x branch creation.